bookwyrm/static/css/bookwyrm.css

```diff
@@ -202,25 +194,85 @@ html {
     min-height: 96px;
 }
 
-/* --- QUOTES --- */
-.quote blockquote {
+/** Statuses: Quotes
+ *
+ * \e906: icon-quote-open
+ * \e905: icon-quote-close
+ *
+ * The `content` class on the blockquote allows to apply styles to markdown
+ * generated HTML in the quote: https://bulma.io/documentation/elements/content/
+ *
+ * ```html
+ * <div class="quote block">
+ *     <blockquote dir="auto" class="content mb-2">
+ *         User generated quote in markdown…
+ *     </blockquote>
+ *
+ *     <p> — <a…>Book Title</a> by <a…class="author">Author</a></p>
+ * </div>
+ * ```
+ ******************************************************************************/
+
+.quote > blockquote {
     position: relative;
     padding-left: 2em;
 }
 
-.quote blockquote::before,
-.quote blockquote::after {
+.quote > blockquote::before,
+.quote > blockquote::after {
     font-family: 'icomoon';
     position: absolute;
 }
 
-.quote blockquote::before {
+.quote > blockquote::before {
     content: "\e906";
     top: 0;
     left: 0;
 }
 
-.quote blockquote::after {
+.quote > blockquote::after {
     content: "\e905";
     right: 0;
 }
+
+/** Animations and transitions
+ ******************************************************************************/
+
+@keyframes beating {
+    0% { font-size: 100%; }
+    25% { font-size: 125%; }
+    100% { font-size: 75%; }
+}
+
+@keyframes turning {
+    from { transform: rotateZ(0deg); }
+    to { transform: rotateZ(360deg); }
+}
+
+.is-processing .icon-heart::before {
+    animation: beating 0.5s infinite alternate ease-in-out;
+}
+
+.is-processing .icon-boost::before {
+    animation: turning 0.5s infinite ease-in-out;
+}
+
+@media (prefers-reduced-motion: reduce) {
+    .is-processing .icon::before {
+        animation-duration: 1.5s;
+    }
+}
```

Comment:
Why do you think they should be combined?

As a general rule, I always found it easier to get around CSS logic and inheritance by keeping similar selectors together instead of grouping huge blocks of rules by media queries.

That’s the [SMACSS](http://www.smacss.com/book/state#media) way, for instance. Grouping based on selectors is closer to a modular or component approach and also documents the logic of the code:

> Yes, this does mean that the media query declaration may (and likely will) get declared multiple times but it also allows for all information about a module to be kept together. Remember, keeping the module information together (especially in its own CSS file) allows for isolated testing of the module and (depending on how you build your application) allows assets such as modularized templates and CSS to be loaded after the initial page has been loaded.

This way is also easier to deal with when working with pre-processors.